src/Microsoft.DotNet.Configurer/NuGetCachePrimer.cs

```diff
@@ -93,8 +103,8 @@ private bool RestoreTemporaryProject(string extractedPackagesArchiveDirectory, s
 
         private bool RunCommand(string commandToExecute, IEnumerable<string> args, string workingDirectory)
         {
-            var command = _commandFactory
-                .Create(commandToExecute, args)
+            var command = Command
```

Comment:
Instead of using Command directly, you should probably use a factory that just invokes it instead. So that you can unit test the behavior.